### DISCUSSION TITLE
Putty software shows error

### DISCUSSION THREAD

**Discussion by Anjumvj** [learndevopsonline/learndevopsonline#1649] [Q&A]
[
![Screenshot 2023-12-01 125444](https://github.com/learndevopsonline/learndevopsonline/assets/130336724/59df7f06-a508-4553-bd6e-434ce8897858)
](url)

**Comment by sreedharm07** [comment]
Check security groups 
It should be attached to instances and it should be with allow all

**Reply by Anjumvj** [reply]
![Screenshot 2023-12-01 133234](https://github.com/learndevopsonline/learndevopsonline/assets/130336724/f0526718-d243-4e8c-b0b6-f6cee7290b7a)
![Screenshot 2023-12-01 125444](https://github.com/learndevopsonline/learndevopsonline/assets/130336724/c7bff133-c5dc-4f91-b41f-651e31cc4a27)

**Reply by sreedharm07** [reply]
Check internet gateway

**Reply by Anjumvj** [reply]
Is this issue due to the wifi network in office? At home it was working smoothly

**Reply by sreedharm07** [reply]
Ok
